tests/unit/test_finder.py

```diff
@@ -234,7 +234,7 @@ def test_link_sorting(self) -> None:
         )
         sort_key = evaluator._sort_key
         results = sorted(links, key=sort_key, reverse=True)
-        results2 = sorted(reversed(links), key=sort_key, reverse=True)
+        results2 = sorted(links, key=sort_key, reverse=True)
```

Comment:
I _think_ `reversed` here is intended, otherwise this line is just identical to the previous one. It’s likely for testing sort stability. Same for other changes in this file.

Reply:
My sense is that `ruff` has it right in this instance.
```
>>> import random
>>> from string import ascii_letters
>>> for _ in range(100):
...     s = ''.join(random.sample(ascii_letters, len(ascii_letters)))
...     print(s)
...     assert sorted(reversed(s), reverse=True) == sorted(s, reverse=True)
...
```

Reply:
Ruff is correct, but this depends on Link implementing ordering correctly, which I think is what this line intends to test against.